client/scripts/DrawTogether.js

```diff
@@ -1295,6 +1326,262 @@ DrawTogether.prototype.handlePaintSelection = function handlePaintSelection (eve
 	}.bind(this));
 };
 
+// Send null for unchanging value
+DrawTogether.prototype.updateIndividualFavoriteDom = function updateIndividualFavoriteDom(newX, newY, newName, newOwner, element) {
+	if(newX !== null)
+		element.dataset.x = newX;
+	if(newY !== null)
+		element.dataset.y = newY;
+	if(newName !== null)
+		element.dataset.name = newName;
+	if(newOwner !== null)
+		element.dataset.owner = newOwner;
+	
+	var coordinateButton = element.getElementsByClassName("fav-coor-button")[0];
```

Comment:
You should not get things by classname or id, because it has to be possible to create multiple drawTogether instances in 1 context. You should put these elements in some kind of variable that you can access or use the this.favContainer access methods to search for the button.

(For example you can run trough its children or you can just save the button somewhere when you create it).